rbx_binary/src/deserializer.rs

```diff
@@ -474,7 +474,27 @@ impl<R: Read> BinaryDeserializer<R> {
             }
             Type::Ray => {}
             Type::Faces => {}
-            Type::Axes => {}
+            Type::Axes => match canonical_type {
+                VariantType::Axes => {
+                    for referent in &type_info.referents {
+                        let instance = self.instances_by_ref.get_mut(referent).unwrap();
+                        let value = chunk.read_u8()?;
+                        let rbx_value =
+                            Variant::Axes(Axes::from_bits(value).unwrap_or(Axes::empty()));
```

Comment:
If there are unnecessary bits set here, this will silently assign an empty set of axes. We should raise an error here instead!

If we do want to keep this fallback to default axes, you can use `.unwrap_or_else(Axes::empty)`, which would defer the `Axes::empty()` call I believe.

Reply:
Should I add a new variant of `InnerError` for an error here, e.g. `InnerError::InvalidPropData`?

Reply:
Yeah, that would work for me. Similar to the other errors we have right now, making them hold strings describing which instance, which property, what we expected, and what we got would be great.